### ISSUE TITLE
[Bug][Tabs Tray]: Synchronized tabs are barely visible after the sync process

### ISSUE THREAD

**Issue by AdinaPetridean** [mozilla-mobile/fenix#26088]
**Preconditions:** 
Make sure you are signed in with the same account on desktop, and another mobile device.
Make sure you have a few tabs open on your desktop.

### Steps to reproduce:

1. From synced tabs tray view tap the "Sign in to sync" button.
2. From the "Turn on Sync" page tap on "Ready to scan" button.
3. Confirm the pairing on both devices.
4. Verify the synced tabs tray.

### Expected behaviour

The synced tabs tray contains a list of devices with the opened tabs that are clearly visible from each device.

### Actual behaviour

The synchronized tabs are barely visible after the synchronization process

### Device name

Tablet Sony Xperia 

### Android version

Android 6.0.1

### Firefox release type

Firefox Nightly

### Firefox version

 Debug Build: 1.0.2229

### Additional information

Link debug build: https://firefox-ci-tc.services.mozilla.com/tasks/UCQAnJ1cSVG5_mUcjOUuxA
 
QA request: https://mozilla-hub.atlassian.net/browse/QA-1561

Please note this issue is also reproducible on OnePlus 6T (Android 9)

![Synchronized tabs](https://user-images.githubusercontent.com/108326843/179681676-071fff5e-5bd4-4f66-b43c-1d10bb62ebe0.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21052)

**Comment by MozillaNoah** [comment]
Hi, @AdinaPetridean, another dev and I haven't had any luck reproducing this. Would you be willing to check if this is reproducible on the build generated in this feature's [pull request](https://github.com/mozilla-mobile/fenix/pull/25895) last week or in the current Nightly build?

**Comment by MozillaNoah** [comment]
I was able to reproduce this on `main` and discovered that this bug is a duplicate of https://github.com/mozilla-mobile/fenix/issues/22558. Closing this in favor of that and to unblock https://github.com/mozilla-mobile/fenix/pull/25895.
